dev-site.yml

```diff
@@ -19,7 +19,7 @@ asciidoc:
     - ./lib/tab-block.js
 ui:
   bundle:
-    url: https://github.com/redhat-developer-demos/rhd-tutorial-ui/releases/download/v0.1.9/ui-bundle.zip
```

Comment:
Updated the UI bundle to improve copy/paste UX.

*BEFORE*
<img width="838" alt="Screenshot 2022-09-08 at 4 13 18 PM" src="https://user-images.githubusercontent.com/1303687/189244589-e83ddf49-b2ff-4f24-b7b2-a5f8c08b5f06.png">
*AFTER*
<img width="833" alt="Screenshot 2022-09-08 at 4 13 31 PM" src="https://user-images.githubusercontent.com/1303687/189244582-6827739d-07f6-4e8c-bf4b-b7e8aa3f15d9.png">